.github/CONTRIBUTING.md

```diff
@@ -14,18 +16,22 @@ I will grant you commit access if you send quality pull requests.
 **Prepare** by installing and migrating the database:
 
 1. Clone this repo
-1. Run `bundle install`
-1. Run `bundle exec rake db:migrate`
-1. Run `RAILS_ENV=test bundle exec rake db:migrate`
+2. Run `bundle install`
+3. Run `bundle exec rake db:migrate`
+4. Run `RAILS_ENV=test bundle exec rake db:migrate`
 
 Now your environment is ready to run tests.
 
-To run the full **test suite** with the [guard](https://github.com/guard/guard) test runner:
+5. To run the full **test suite** with the [guard](https://github.com/guard/guard) test runner:
```

Comment:
I don't think this is a new step, it's optional and separated from the main instructions